lib/connection.c

```diff
@@ -1344,4 +1344,16 @@ int64 get_retry_timeout(SF_CONNECT *sf)
 int8 get_login_retry_count(SF_CONNECT *sf)
 {
   return (int8)get_less_one(sf->retry_on_connect_count, sf->retry_count);
+}
+
+sf_bool end_with(char* a, char* b)
```

Comment:
Please use descriptive names for arguments and variables

Reply:
Updated it.